narwhals/_ibis/namespace.py

```diff
@@ -51,8 +51,12 @@ def _function(self, name: str, *args: ir.Value | PythonLiteral) -> ir.Value:
     def _lit(self, value: Any) -> ir.Value:
         return lit(value)
 
-    def _when(self, condition: ir.Value, value: ir.Value) -> ir.Value:
-        return ibis.cases((condition, value))
+    def _when(
+        self, condition: ir.Value, value: ir.Value, otherwise: ir.Expr | None = None
+    ) -> ir.Value:
+        if otherwise is None:
+            return ibis.cases((condition, value))
+        return ibis.cases((condition, value), else_=otherwise)
```

Comment:
looks like we don't (yet) have a test case which covers this for ibis yet, so you could add `# pragma: no cover` to the end this line (59)